PugSharp.Match/Match.cs

```diff
@@ -43,6 +43,8 @@ public class Match : IDisposable
 
     public MatchInfo MatchInfo { get; private set; } = default!;
 
+    public event EventHandler<MatchFinalizedEventArgs>? MatchFinalized;
```

Comment:
Nehmen wir jetzt doch eventhandler statt matchcallback?

Reply:
MatchCallback gibt es gerade nicht mehr. Ich wollte die Kapselung in die Richtung nicht so gern wieder einbauen

Reply:
Finde ich gut!